packages/nexus-sdk/src/types.ts

```diff
@@ -14,6 +15,10 @@ export type Context =
   | (string | { [key: string]: string })[]
   | { [key: string]: string };
 
+// TODO: find the real way, I know this is cheating...
+const nexus = createNexusClient({ uri: '' });
```

Comment:
This bothers me, either the uri param should be optional and default to `''`, so you do not have to pass it here, or it should be enforced and never be empty.

Reply:
You're right, the comment is unclear. By doing this, it dynamically generated the types for TS